resources/assets/lib/notifications/worker.ts

```diff
@@ -39,9 +39,9 @@ export default class Worker implements DispatchListener {
   @observable waitingVerification = false;
 
   @observable private firstLoadedAt?: Date;
-  private timeout: Record<string, number> = {};
-  private xhr: Record<string, JQueryXHR> = {};
-  private xhrLoadingState: Record<string, boolean> = {};
+  private timeout: Partial<Record<string, number>> = {};
+  private xhr: Partial<Record<string, JQueryXHR>> = {};
+  private xhrLoadingState: Partial<Record<string, boolean>> = {};
```

Comment:
These should should probably have explicit keys listed instead of being `Partial`. Or changed since the only thing in them seems to be `loadMore` 🤔

Reply:
Even with explicit keys those will still be partial (or explicit allow undefined/null) otherwise they must be initialized with dummy value (which seems worse).